components/RecordMemory.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import useAuth from '../utils/useAuth'
+// import PropTypes from 'prop-types'
+import { Row, Col, Container, Button } from 'react-bootstrap/'
+import { useRouter } from 'next/router'
+
+const RecordMemory = () => {
+  useAuth()
```

Comment:
I think you'll still need something like this to make the auth work as expected:

const [, token] = useAuth()

if (!token) {
return <p>Loading</p>
}

Reply:
I _think_ I've got that now

Reply:
Yep, all looks good